modules/ROOT/pages/index.adoc

```diff
@@ -10,7 +10,7 @@ to write directly to object store, use its REST API, and write and read using
 the Object Store connector. See the xref:osv2-faq.adoc[FAQ] for
 Object Store information.
 
-NOTE: Mule 4 only supports Object Store v2.
+NOTE: CloudHub only supports Object Store v2 for creating Mule 4 applications. This does not affect on-premises use of Mule 4. Object Store v2 only stores data in the cloud. On-premises use of Mule stores data locally to a persistent data store.
```

Comment:
CH supports osv2 and in memory os.

Reply:
Yes options are OSv2 and in-memory for Mule 4.